sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/__init__.py

```diff
@@ -5,7 +5,11 @@
 # -------------------------------------------------------------------------
 
 from ._azureappconfigurationprovider import load, AzureAppConfigurationProvider
-from ._models import AzureAppConfigurationKeyVaultOptions, SettingSelector
+from ._models import (
+    AzureAppConfigurationKeyVaultOptions,
+    SettingSelector,
+    AzureAppConfigurationRefreshOptions,
```

Comment:
We shouldn't add an object for this.

Reply:
Not `**kwargs` specifically, but we should flatten it out to separate keyword-only arguments.